samples/snippets/csharp/concepts/linq/LinqSamples/SubqueryOnGroup.cs

```diff
@@ -39,10 +39,13 @@ public static void SubqueryOnGroup2()
                 .Select(studentGroup => new
                 {
                     Level = studentGroup.Key,
-                    HighestScore = studentGroup.Select(student2 => student2.ExamScores.Average()).Max()
+                    HighestScore = studentGroup
+                        //.Select(student2 => student2.ExamScores.Average())    // fold into single method call
+                        //.Max()
+                        .Max(student2 => student2.ExamScores.Average())
```

Comment:
Removing the commented out code. It's more clear for the reader:

```suggestion
                    HighestScore = studentGroup.Max(student2 => student2.ExamScores.Average())
```